### PR TITLE
Driver & Constructor Standings from Server

### PR DESCRIPTION
**Pull request by Lombardoc4**
Fetching data from the server and populating the revised look of the driver and constructor standings.

### CODE REVIEW

File: src/app/lib/utils.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { faker } from '@faker-js/faker';
+import axios from 'axios';
```
Comment (borolepratik) [inline src/app/lib/utils.tsx:2]: @Lombardoc4 is there any particular reason to use axios over fetch?

File: src/app/lib/utils.tsx [hunk 2]
```diff
@@ -131,54 +77,89 @@ const dataConfig: IDataConfigs = {
...
     })),
+    season: 0,
+    round: 0,
   },
 };
 
-export const fetchAPI = async (endpoint: string) => {
-  const server = document.body.classList.contains('server');
-  const dummy: string[] | ISchedule[] | false =
+const serverURL = 'http://127.0.0.1:8081';
```
Comment (borolepratik) [inline src/app/lib/utils.tsx:121]: can we move this to a dedicated constants file?

**Comment by borolepratik** [comment]
looks good to me.

**Review by borolepratik — APPROVED** [review]

File: src/app/lib/utils.tsx (hunk 1 above)
Comment (Lombardoc4) [reply src/app/lib/utils.tsx:2]: A little more comfortability in the moment. Going to refactor this away along with faker in next rev to start to properly handle server errors.

**Review by borolepratik — APPROVED** [review]

**Pull request merged**